draft-ietf-httpbis-digest-headers.md

```diff
@@ -1220,27 +1196,13 @@ Note that a `204 No Content` response without content but with the same
 In error responses, the representation-data does not necessarily refer to the
 target resource. Instead, it refers to the representation of the error.
 
-In the following example a client attempts to patch the resource located at
-/books/123. However, the resource does not exist and the server generates a 404
-response with a body that describes the error in accordance with {{?RFC7807}}.
+In the following example, a client sends the same request from {{fig-patch}} to
+patch the resource located at /books/123. However, the resource does not exist
+and the server generates a 404 response with a body that describes the error in
+accordance with {{?RFC7807}}.
 
 The response `Digest` field-value is computed on this enclosed representation.
 
-Request:
-
-~~~ http-message
-PATCH /books/123 HTTP/1.1
-Host: foo.example
-Content-Type: application/merge-patch+json
-Accept: application/json
-Accept-Encoding: identity
-Digest: sha-256=bWopGGNiZtbVgHsG+I4knzfEJpmmmQHf7RHDXA3o1hQ=
-
-{"title": "New Title"}
-~~~
```

Comment:
Was this deletion intentional?

Reply:
Oh I see, you ref to the previous example - clever.

Reply:
Yes, it was identical to the previous request, so I didn't see a need to repeat it.